### PR TITLE
CUMULUS-3006 - removing executions terraform code

### PR DESCRIPTION
**Pull request by nasamoduyebo**
**Summary:** Summary of changes

Addresses [CUMULUS-3006: Remove Dynamodb Executions table](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3006)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Review by charleshuang80 — COMMENTED** [review]
In general the PR looks good to me. I have 2 questions I think require asking someone more knowledgeable. Also I don't know that I would feel confident to be the only approval when it is ready.

File: tf-modules/data-persistence/terraform.tfvars.example [hunk 1]
```diff
@@ -14,7 +14,6 @@ elasticsearch_config = {
 }
 enable_point_in_time_tables = [
   "CollectionsTable",
-  "ExecutionsTable"
```
Comment (Jkovarik) [inline tf-modules/data-persistence/terraform.tfvars.example:17]: Good catch!

File: tf-modules/ingest/lambda-functions.tf [hunk 2]
```diff
@@ -237,7 +237,6 @@ resource "aws_lambda_function" "sf_sqs_report_task" {
     variables = {
       CUMULUS_MESSAGE_ADAPTER_DIR = "/opt/"
       stackName                   = var.prefix
-      ExecutionsTable             = var.dynamo_tables.executions.name
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply tf-modules/ingest/lambda-functions.tf:240]: Based on our huddle convo - I'm not clear that this task has needed this variable for some time - in either case, the lambda is still needed.

File: tf-modules/archive/ingest-reporting.tf [hunk 3]
```diff
@@ -37,15 +37,6 @@ data "aws_iam_policy_document" "publish_executions_policy_document" {
     resources = [aws_sqs_queue.publish_executions_dead_letter_queue.arn]
   }
 
-  statement {
```
[the remark this answers is not in the document]
Comment (Jkovarik) [reply tf-modules/archive/ingest-reporting.tf:40]: I think it's fine to remove this (and anything that references it) - the 'publish_executions' lambda is going to be removed in https://bugs.earthdata.nasa.gov/browse/CUMULUS-2994 and/or once this is merged https://github.com/nasa/cumulus/pull/3276/files

You can probably check with @marchuffnagle  to see where that's at.

**Review by Jkovarik — APPROVED** [review]
This looks good, great work! 

Please see comments re: @marchuffnagle 's PR that has some related updates ,as well as #3280 as a suggestion for cleanup that's probably appropriate to this PR.

**Comment by Jkovarik** [comment]
@nasamoduyebo  I didn't see an update comment - did you see https://github.com/nasa/cumulus/pull/3280 and decide not to include those updates in this PR?

**Comment by nasamoduyebo** [comment]
> @nasamoduyebo I didn't see an update comment - did you see #3280 and decide not to include those updates in this PR?

@Jkovarik Yikes no i did see this i thought you would be merging these changes in separately. this is a valid part of 3006 that should have been included

**Comment by nasamoduyebo** [comment]
@Jkovarik What i can do is take over this PR and make sure builds are passing after merging the latest code into this PR

**Pull request merged**